standalone/ansible/roles/logging/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
This should rotate for **all** boxes correct? Including HAProxy, database etc.?

Reply:
This setup is for the rails app only i.e webservers and sidekiq. Do we want to logrotate/ship HAProxy and database logs as well?

Reply:
Yes, we should. Those logs also grow over time and should be archived.